mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+                     Whether column statistic needs to be strictly lesser than ('<') the maximum, or if equality is allowed.
+                     Only relevant if a maxValue has been defined. Default = false.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'rowConditionExpectation'
+               description: |
+                 Table rule which evaluates whether each row passes the specified condition.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'sqlExpression'
+                   description: |
+                     The SQL expression.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'tableConditionExpectation'
+               description: |
+                 Table rule which evaluates whether the provided expression is true.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'sqlExpression'
+                   description: |
+                     The SQL expression.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dataProfileSpec'
+    description: |
+      DataProfileScan related setting.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dataQualityResult'
+    output: true
+    description: |
+      The result of the data quality scan.
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'passed'
+        description: |
+          Overall data quality result -- true if all rules passed.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'dimensions'
+        description: |
+          A list of results at the dimension level.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'passed'
+            description: |
+              Whether the dimension passed or failed.
+              item_type: !ruby/object:Api::Type::NestedObject
+      - !ruby/object:Api::Type::Array
+        name: 'rules'
+        description: |
+          A list of all the rules in a job, and their results.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::NestedObject
+              name: 'rule'
+              description: |
+                The rule specified in the DataQualitySpec, as is.
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'column'
+                  description: |
+                    The unnested column which this rule is evaluated against.
+                - !ruby/object:Api::Type::Boolean
+                  name: 'ignoreNull'
+                  description: |
+                    Rows with null values will automatically fail a rule, unless ignoreNull is true. In that case, such null rows are trivially considered passing. Only applicable to ColumnMap rules.
+                - !ruby/object:Api::Type::String
+                  name: 'dimension'
+                  required: true
```

Comment:
as an output, this shouldn't be required, right?

Reply:
+1